src/scout_apm/core/tracked_request.py

```diff
@@ -34,6 +34,9 @@ class TrackedRequest(object):
         "n_plus_one_tracker",
     )
 
+    # Stop adding new spans at this point, to avoid exhausting memory
+    MAX_COMPLETE_SPANS = 100000
```

Comment:
Let's lower this to 1500 to match our Ruby agent.

Reply:
Done